src/esp/physics/bullet/BulletPhysicsManager.cpp

```diff
@@ -274,6 +274,7 @@ bool BulletPhysicsManager::attachLinkGeometry(
     const std::shared_ptr<io::URDF::Link>& link,
     gfx::DrawableGroup* drawables,
     const std::string& lightSetup) {
+  bool reqLighting = (lightSetup != esp::NO_LIGHT_KEY);
```

Comment:
This will force component assets for articulated objects to be flat shaded if no-lights is specified as the lighting setup.